### DISCUSSION TITLE
High CPU usage...

### DISCUSSION THREAD

**Discussion by leonidcan** [FlareSolverr/FlareSolverr#814] [Q&A]
I only have 10-20 active sessions and CPU usage is high (average requests are just ~10-20 per minute total). 
Is this normal? Running in docker on VMs. I had 4 cores per VM and most of the time the load was close to 100% with some request actually time out. I added few more threads and it's better but sometimes spikes to 100% anyway.
Is it cloudflare detecting me or something else? Is it normal? I see a lot of chrome instances taking most of the CPU.

**Comment by ngosang** [comment]
If you have 20 sessions (20 windows open in Chrome) it's normal.
